pkg/kubelet/dockershim/libdocker/client.go

```diff
@@ -27,16 +27,21 @@ import (
 )
 
 const (
+	// MinimumDockerAPIVersion is the lower docker version acceptable
 	// https://docs.docker.com/engine/reference/api/docker_remote_api/
-	// docker version should be at least 1.13.1
+	// docker version should be at least this high
```

Comment:
Just double checking: Does the git history show that `MinimumDockerAPIVersion` used to be 1.13.1, but someone updated it to 1.26.0 and forgot to update the comment?

Reply:
It was updated in this (f62cc81934634433eb8c7dbfc5bf755247a8efeb). It seems that 1.13.1 is the docker version while the 1.26.0 is the API version. I'll update the comment so this distinction is a little more clear to the reader.